django_filters/fields.py

```diff
@@ -92,6 +92,7 @@ def __init__(self, field, lookup_choices, *args, **kwargs):
         }
         widget = LookupTypeWidget(**defaults)
         kwargs['widget'] = widget
+        kwargs['help_text'] = field.help_text
```

Comment:
Out of curiosity - what effect does this line have? It looks like the `MultiValueField` is using the same `help_text` as the internal field instance. Does the internal field not render it's `help_text` or something?